packages/back-end/src/models/WebhookModel.ts

```diff
@@ -39,7 +41,18 @@ type WebhookDocument = mongoose.Document & WebhookInterface;
 const WebhookModel = mongoose.model<WebhookInterface>("Webhook", webhookSchema);
 
 function toInterface(doc: WebhookDocument): WebhookInterface {
-  return omit(doc.toJSON<WebhookDocument>(), ["__v", "_id"]);
+  const oldDoc = doc.toJSON<WebhookDocument>();
+  const newDoc = omit(oldDoc, ["__v", "_id", "sendPayload"]);
+  if (!oldDoc.payloadFormat) {
```

Comment:
Should you also check that `sendPayload` is not set?

Reply:
don't think so. If the payloadFormat exists, it should take precidence